docs/source/programming.md

```diff
@@ -0,0 +1,123 @@
+# Programming a neutral-atom Quantum Computer
+
+## Introduction
+
```

Comment:
how about having in every page the list of "what will you learn in this page".

This can help the user a lot and probably be very helpful for indexing pages for search engines, like typing in google "rydberg" would bring us up in the search. Wdyt?

Reply:
Do you really think it will help for page indexation in Google ? If you have documentation regarding this I am all hear 
I have not found better than this https://blog.readthedocs.com/seo-for-technical-docs/
But they don't mention this. However, I guess it can help the user so let's try to write something indeed.

Reply:
So my starting point was to facilitate the users and I think also the structure of the documentation. In principle by just looking at this first box for each page we should be able to see what we cover and what we don't.
The indexing was a bit speculative: I don't know any detail about how this could work but at the same point I am pretty confident that this should help in this direction.
Probably the best people to ask as frontend people. I iwll drop a message askingif anyone has good advice in this direction too